src/assets/scss/components/_category.scss

```diff
@@ -28,23 +28,11 @@
       height: 100%;
     }
     &.selected {
-      background-color: $primary-color;
-      color: #FFF !important;
-      &:hover, &:active {
-        color: $primary-color !important;
-      }
+      background-color: mix(#FFF, $primary-color, 50);
     }
   }
-  &__heading {
-    flex-basis: 33%;
-  }
-  &__current {
-    color: mix(#000, #FFF, 50);
-    text-align: right;
-    width: 100%;
-  }
 }
 
-.green-expansion-panel {
-  border: 1px solid $primary-color;
+.category-expansion-panel {
+  border: 1px solid #eee;
```

Comment:
Should #eee be a color variable instead?

Reply:
Good point - have changed